### ISSUE TITLE
Remove "new" calls to Ember.A()

### ISSUE THREAD

**Issue by notmessenger** [softlayer/sl-ember-components#720]
* addon/components/sl-calendar.js
* tests/dummy/app/controllers/browsers.js
* tests/unit/components/sl-grid-test.js
* tests/unit/components/sl-menu-test.js

UPDATE: Remove this from all of the dummy application code as well.

**Comment by notmessenger** [comment]
https://github.com/joshforisha/sl-eslint/issues/9 plays into this

**Comment by joshforisha** [comment]
Why?

**Comment by notmessenger** [comment]
Because Ember.Array is not an object to be made an instance of.  It is a method that returns an array.

http://emberjs.com/api/classes/Ember.html#method_A
https://github.com/emberjs/ember.js/blob/v2.0.1/packages/ember-runtime/lib/system/native_array.js#L137-L140

**Comment by azizpunjani** [comment]
@notmessenger please assign to me, working on this.

**Comment by notmessenger** [comment]
@azizpunjani Also remove this from all of the dummy application code (there are many instances of this).  This was not part of the original task description but has been identified as needing to be done as well.

**Comment by azizpunjani** [comment]
@notmessenger this has been done, we are now getting errors via eslint after removing "new" https://travis-ci.org/softlayer/sl-ember-components/jobs/78290883#L545. How should we proceed?

**Comment by joshforisha** [comment]
@azizpunjani I have PR #742 that should take care of that.

**Comment by azizpunjani** [comment]
@notmessenger once #742 is merged in, I will merge it into the branch I created and test again to make sure eslint errors are not thrown anymore.
